cluster/vmpreset-windows.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows2012r2-preset
+  selector:
+    matchLabels:
+      flavor: windows2012r2
```

Comment:
I'd suggest to use `os-short-id` as the key name, and the corresponding libosinfo shortid `win2k12r2` as the value in order to align on the os terminology and values.